pageserver/src/tenant.rs

```diff
@@ -1792,6 +1802,8 @@ impl Tenant {
             .unwrap()
             .success(&CIRCUIT_BREAKERS_UNBROKEN);
 
+        tracing::info!(?has_pending_task, "done");
```

Comment:
remove the info! before merging?

Reply:
Same as above ("temp:" prefixed commit, or at least it meant to be :))